.travis.yml

```diff
@@ -109,6 +109,16 @@ jobs:
         GOAL="install"
         BITCOIN_CONFIG="--enable-zmq --disable-wallet --with-gui=qt5 CPPFLAGS=-DDEBUG_LOCKORDER --with-sanitizers=thread --disable-hardening --disable-asm CC=clang CXX=clang++"
 
+    - stage: test
+      name: ''x86_64 Linux  [GOAL: install]  [trusty]  [no depends, only system libs]''
```

Comment:
Looks like extra quotes around the name.

Reply:
Ah yeah, thanks.